nucliadb/nucliadb/writer/resource/field.py

```diff
@@ -413,3 +419,7 @@ async def parse_conversation_field(
         convs.messages.append(processing_message)
 
     toprocess.conversationfield[key] = convs
+
+
+def is_external_file(field: models.FileField) -> bool:
+    return field.file.uri is not None
```

Comment:
I think this should be part of the model definition (as a property for example). To avoid spreading implementation details on other parts of the code